src/Home/Home.tsx

```diff
@@ -3,6 +3,10 @@ import { useContext } from "react";
 import { Journal, JournalEntry, JournalEntries, gen_data } from "../journal";
 import { JournalContext, useJournal } from "../JournalContext"
 
+// ceverett, Sprint in Progress #2: (calendar functionality)
+import Calendar from 'react-calendar'; // "npm install,list react-calendar"
```

Comment:
we need to have the package saved in the `package.json`. when we want to have packages saved we can run `npm install --save <package>` and if we want to save development dependencies then we can run `npm install --save-dev <package>`